.mocharc.yml

```diff
@@ -1 +1 @@
-require: 'ts-node/register'
+require: 'ts-node/register/transpile-only'
```

Comment:
Hmm, is this required? Not using the full `ts-node/register` breaks source-mapping in the WebStorm mocha plugin - so to debug I always have to turn it back on 😞

Reply:
Contrib repo is using it to avoid type issues with older versions of an instrumented package when running test-all-versions. A side effect is slightly faster test since type checking is skipped but it's okay to rollback to the previous config.

Reply:
Done in https://github.com/open-telemetry/opentelemetry-js/pull/5145/commits/a4e30a72c25825de73a0a6315b6f15a0ba07abad

Reply:
thanks 🙂